controllers/admin/AdminImportController.php

```diff
@@ -2517,8 +2517,13 @@ protected function attributeImportOne($info, $default_language, &$groups, &$attr
                                 'Admin.Advparameters.Notification'
                             );
                         }
+                        if (function_exists('mysqli_fetch_all')) {
+                            $my_sql_error = mysqli_error();
+                        } elseif (strpos($pdo->getAttribute(PDO::ATTR_CLIENT_VERSION), 'mysqlnd') !== false) {
+                            $my_sql_error = PDO::errorInfo();
```

Comment:
It will not work. You can have the mysqli extension enable but this is the pdo which is running.
You can check for `getMsgError` with is available in Db instance =)

Reply:
I think it is more safe to let you make a fix. :-)

Reply:
`Db::getInstance()->getMsgError();` wdyt @Quetzacoalt91 ?